frontend/src/bundles/auth/store/slice.ts

```diff
@@ -6,24 +6,41 @@ import {
     type ValueOf,
 } from '~/bundles/common/types/types.js';
 
-import { logout, refreshUser, signIn, signUp, updateUser } from './actions.js';
+import {
+    logout,
+    refreshUser,
+    signIn,
+    signUp,
+    updateUser,
+    uploadAvatar,
+} from './actions.js';
 
 type State = {
     dataStatus: ValueOf<typeof DataStatus>;
     user: UserAuthResponseDto | null;
     isRefreshing: boolean;
+    updateProfile: {
+        avatarUrl: string | null;
+    };
```

Comment:
why do we need this additional state?

Reply:
To save avatarUrl after dispatch uploadAvatar.

After fulfilled action I get this new avatarUrl from state and setValue in form and clear this state